src/pages/permissions/panels/PermissionSelector.tsx

```diff
@@ -1,15 +1,17 @@
 import { Button, useNotify } from "@canonical/react-components";
-import CustomSelect from "../../../components/select/CustomSelect";
+import CustomSelect, {
+  SelectRef,
+} from "../../../components/select/CustomSelect";
```

Comment:
```suggestion
} from "components/select/CustomSelect";
```
Better use the non-relative imports